.changeset/stupid-elephants-trade.md

```diff
@@ -0,0 +1,6 @@
+---
+"@salt-ds/core": minor
+---
+
+Removed `interactable` and `disabled` props from Card
```

Comment:
This is a breaking change and won't be able to be made. We should deprecate these props instead

Reply:
Another way of doing this is to still have InteractableCard and when interactable is present to conditionally render it. Would make things more complicated though